mongodump/prepare_test.go

```diff
@@ -142,6 +143,12 @@ func TestShouldSkipSystemNamespace(t *testing.T) {
 			coll:   "foo",
 			output: true,
 		},
+		{
+			db:       "config",
+			coll:     "foo",
```

Comment:
Can we also add a test for a collection that would be included normally like `chunks` in ConfigCollectionsToKeep.

Reply:
Added